src_assets/common/assets/web/config.html

```diff
@@ -806,17 +806,13 @@ <h1 class="my-4">Configuration</h1>
             id: "advanced",
             name: "Advanced",
           },
-          {
-            id: "sw",
-            name: "Software Encoder",
-          },
           {
             id: "nv",
             name: "NVENC Encoder",
```

Comment:
Should we change this to `Nvidia NVENC Encoder`?

Reply:
I think NVENC is more recognizable (mainly because NVIDIA has used it in marketing material), but I added "NVIDIA" to be consistent.